packages/elements/src/photon-multirx-form/components/AddPrescriptionCard.tsx

```diff
@@ -117,118 +97,81 @@ export const AddPrescriptionCard = (props: {
   };
 
   const handleAddPrescription = async () => {
+    setIsLoading(true);
+
+    // TODO TODO TODO move validation to the prescribe provider
     const keys = Object.keys(validators);
     props.actions.validate(keys);
     const errorsPresent = props.actions.hasErrors(keys);
 
-    const draftedPrescriptions = [...props.store.draftPrescriptions.value];
-    const isPrescriptionAlreadyAdded = isTreatmentInDraftPrescriptions(
-      props.store.treatment?.value?.id,
-      draftedPrescriptions
-    );
-
-    if (isPrescriptionAlreadyAdded) {
+    if (errorsPresent) {
+      setIsLoading(false);
       triggerToast({
         status: 'error',
-        body: 'You already have this prescription in your order. You can modify the prescription or delete it in Pending Order.'
+        body: 'Some items in the form are incomplete, please check for errors'
```

Comment:
are we still checking for any duplicates? looking at the old message it seems like a different error